src/InterventionBackend.php

```diff
@@ -30,6 +35,36 @@ class InterventionBackend implements Image_Backend, Flushable
     const CACHE_DIMENSIONS = 'DIMENSIONS_';
 
     /**
+     * How long to cache each error type
+     *
+     * @config
+     * @var array Map of error type to config.
+     * each config could be a single int (fixed cache time)
+     * or list of integers (increasing scale)
+     */
+    private static $error_cache_ttl = [
+        self::FAILED_INVALID => 0, // Invalid file type should probably never be retried
+        self::FAILED_MISSING => '5,10,20,40,80', // Missing files may be eventually available
```

Comment:
Couldn't `FAILED_MISSING` be an array of integers rather than a comma separated string?
Is it this way for convenience when using `yml` files?